source/codegen/proto_helpers.py

```diff
@@ -168,10 +107,20 @@ def get_enum_parameters(parameter, parameter_name, parameter_type, is_array, use
   })
   return enum_parameters
 
-def get_parameter_type(parameter, is_array, service_class_prefix):
-  if 'grpc_type' in parameter:
-    parameter_type = parameter['grpc_type']
-  else:
-    parameter_type = get_grpc_type_from_ivi(parameter["type"], is_array, service_class_prefix)
-  return parameter_type
 
+def get_parameter_type(parameter):
+  return parameter['grpc_type']
+
+
+def get_parameters(function):
```

Comment:
Pulled out of proto.mako with new support for `has_session_input` where any init method with a `sessionName` param will use that instead of adding a new one.

This allows DAQ to use session_name as task_name.